Orange/tests/sql/test_sql_table.py

```diff
@@ -764,6 +764,20 @@ def test_list_tables_with_schema(self):
             with self.backend.execute_sql_query("DROP SCHEMA IF EXISTS orange_tests CASCADE"):
                 pass
 
+    @dbt.run_on(["postgres"])
```

Comment:
It should be also tested on mssql

```suggestion
    @dbt.run_on(["postgres", "mssql"])
```